mne/io/edf/edf.py

```diff
@@ -204,8 +204,8 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     # make sure events without duration get one sample
                     n_stop = n_stop if n_stop > n_start else n_start + 1
                     if any(stim[n_start:n_stop]):
-                        raise NotImplementedError('EDF+ with overlapping '
-                                                  'events not supported.')
+                        warn('EDF+ with overlapping events'
+                             ' are not fully supported')
```

Comment:
Here the conflict arised from collision of TTL annotations written continuously in the stim channel, and, user annotations inputed sequentially with the recording software.

I noticed the user annotated events are available in `._raw_extras[0]['events']]`.
Can we use the MNE Python annotation system for those ones?